### PR TITLE
RHOAIENG-4214: fixing links

### PR DESCRIPTION
**Pull request by syaseen-rh**
## Description
fixing links

## How Has This Been Tested?
local build

### CODE REVIEW

File: modules/viewing-metrics-for-the-single-model-serving-platform.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ ifdef::self-managed[]
...
-* You are familiar with querying metrics in user-defined projects. See link:https://docs.openshift.com/dedicated/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Dedicated) or link:https://docs.openshift.com/rosa/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Service on AWS).
+* You are familiar with querying metrics in user-defined projects. See link:https://docs.openshift.com/observability/dedicated/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Dedicated) or link:https://docs.openshift.com/observability/rosa/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Service on AWS).
```
Comment (chtyler) [inline modules/viewing-metrics-for-the-single-model-serving-platform.adoc:17]: Maybe it's something I'm doing wrong, but these two links are not working for me.

File: modules/viewing-metrics-for-the-multi-model-serving-platform.adoc [hunk 2]
```diff
@@ -14,7 +14,7 @@ ifdef::self-managed[]
...
-* You are familiar with querying metrics in user-defined projects. See link:https://docs.openshift.com/dedicated/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Dedicated) or link:https://docs.openshift.com/rosa/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Service on AWS).
+* You are familiar with querying metrics in user-defined projects. See link:https://docs.openshift.com/dedicated/observability/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Dedicated) or link:https://docs.openshift.com/rosa/observability/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Service on AWS).
```
Comment (chtyler) [inline modules/viewing-metrics-for-the-multi-model-serving-platform.adoc:17]: tested these links and they work.

File: modules/viewing-metrics-for-the-multi-model-serving-platform.adoc [hunk 3]
```diff
@@ -30,7 +30,7 @@ ifdef::upstream,self-managed[]
...
 endif::[]
 ifdef::cloud-service[]
-. As described in link:https://docs.openshift.com/dedicated/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Dedicated) or link:https://docs.openshift.com/rosa/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Service on AWS), use the web console to run queries for `modelmesh_*` metrics.
+. As described in link:https://docs.openshift.com/dedicated/observability/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Dedicated) or link:https://docs.openshift.com/rosa/observability/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Service on AWS), use the web console to run queries for `modelmesh_*` metrics.
```
Comment (chtyler) [inline modules/viewing-metrics-for-the-multi-model-serving-platform.adoc:33]: tested these links and they work.

File: modules/viewing-metrics-for-the-single-model-serving-platform.adoc [hunk 4]
```diff
@@ -30,7 +30,7 @@ ifdef::upstream,self-managed[]
...
+. As described in link:https://docs.openshift.com/dedicated/observability/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Dedicated) or link:https://docs.openshift.com/rosa/observability/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Service on AWS), use the web console to run queries for `caikit_*`, `tgi_*`, `ovms_*` or `istio_*` metrics.
```
Comment (chtyler) [inline modules/viewing-metrics-for-the-single-model-serving-platform.adoc:33]: tested  these links and they work.

File: modules/viewing-metrics-for-the-single-model-serving-platform.adoc (hunk 1 above)
Comment (syaseen-rh) [reply modules/viewing-metrics-for-the-single-model-serving-platform.adoc:17]: Oh, I see what I did here; the "observability" should be after Rosa/dedicated. Fixing.

**Comment by aduquett** [comment]
LGTM

**Pull request merged**